src/test/resources/liquibase/harness/change/expectedSql/cloudspanner/dropAllForeignKeyConstraints.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE posts ADD CONSTRAINT fk_posts_authors_test_1 FOREIGN KEY (author_id) REFERENCES authors (id)
```

Comment:
This looks weird. The test says `dropAllForeignKeyConstraints`, but this script adds foreign keys.